src/App.js

```diff
@@ -1,29 +1,6 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+  <Calculator />;
```

Comment:
- [ ] KIndy create a class-based component for your  App component,this is a project requirement
![image](https://user-images.githubusercontent.com/61408860/199534771-31773235-0917-481d-a08e-22b5355a0eb8.png)